ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/execution/StaticQueryExtecutorTest.java

```diff
@@ -37,10 +38,13 @@
 import org.mockito.junit.MockitoJUnitRunner;
 
 @RunWith(MockitoJUnitRunner.class)
-public class QueryValidatorTest {
+public class StaticQueryExtecutorTest {
```

Comment:
typo 'StaticQueryExtecutorTest'

Reply:
Always good when a one line code change as a mistake in it ;)

Thanks for catching.